src/instruments/qml/MuseScore/Instruments/internal/FamilyView.qml

```diff
@@ -107,7 +120,15 @@ Item {
         }
 
         delegate: ListItemBlank {
-            isSelected: privateProperties.currentGroupIndex === index
+            id: item
+
+            isSelected: prv.currentGroupIndex === index
+
+            navigation.name: modelData.name
+            navigation.panel: navPanel
+            navigation.row: 2 + model.index
```

Comment:
Please, explain why you use 2 here

Reply:
2 this is next row after row 1 (navigation.name: "familiesBox")